app/models/health/patient.rb

```diff
@@ -1008,7 +1008,8 @@ def assigned_nurse_care_manager_in_agency?
     end
 
     def available_user_ids
-      Health::AgencyUser.where(agency_id: health_agency.id).pluck(:user_id)
+      user_ids = Health::AgencyUser.where(agency_id: health_agency.id).pluck(:user_id)
+      User.where(id: user_ids).merge(User.active).pluck(:id)
```

Comment:
I wanted to do something like `Health::AgencyUser.joins(:user).merge(User.active)` as an `active` scope on `AgencyUser`, but, I'm unable to run that join. I think because the users table is in a different database.

Reply:
Yep, you are right... I get bitten by forgetting that regularly :-)
Dunno which is cleaner, but you could flip this around as:
```
Health::AgencyUser.where(agency_id: health_agency.id, user_id: User.active.pluck(:id)).pluck(:user_id)
```
Just to keep it focused on the Health domain?

Reply:
Nice, I like that 👍  updated